src/sparsezoo/api/exceptions.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from functools import wraps
+from typing import Callable, Optional
+
+from requests.exceptions import HTTPError
+from requests.models import Response
+
+
+logger = logging.getLogger(__name__)
+
+
+class InvalidQueryValueError(Exception):
+    def __init__(self, error_message: Optional[str] = None, *args, **kwargs):
+        super().__init__(
+            error_message,
+            *args,
+            **kwargs,
+        )
```

Comment:
Is this really needed? I'd think based on the content of this method that we wouldn't need to define an `__init__` method here at all

Reply:
without init the error will be 
```
(.venv) george@Georges-MacBook-Pro sparsezoo % python3 '/Users/george/neuralmagic/sparsezoo/scratch/graohql.py'
Traceback (most recent call last):
  File "/Users/george/neuralmagic/sparsezoo/scratch/graohql.py", line 15, in <module>
    m = gql.fetch(operation_body=operation_body, fields=fields, arguments=arguments)
  File "/Users/george/neuralmagic/sparsezoo/src/sparsezoo/api/graphql.py", line 47, in fetch
    response_objects = self.make_request(
  File "/Users/george/neuralmagic/sparsezoo/src/sparsezoo/api/exceptions.py", line 40, in inner_function
    return fn(*args, **kwargs)
  File "/Users/george/neuralmagic/sparsezoo/src/sparsezoo/api/graphql.py", line 81, in make_request
    validate_graphql_response(response=response, query_body=query.query_body)
  File "/Users/george/neuralmagic/sparsezoo/src/sparsezoo/api/exceptions.py", line 56, in validate_graphql_response
    raise InvalidQueryValueError(
TypeError: InvalidQueryValueError() takes no keyword arguments
(.venv) george@Georges-MacBook-Pro sparsezoo % 
```

Reply:
ah i got it